base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -692,6 +692,7 @@ where
             &commitment,
             single_round_sender_data.amount,
         )?;
+        let minimum_value_promise = MicroTari::zero();
```

Comment:
This is dangerous as well, perhaps we should ad this to `TransactionSenderMessage`